Cargo.toml

```diff
@@ -53,15 +53,18 @@ nonempty = "0.10"
 percent-encoding = "2"
 prometheus = "0.13"
 ref-cast = "1"
-reqwest = "0.11"
+reqwest = "0.12"
```

Comment:
Can we bump this separately before the PR goes in? Good not to mix up functional and non-functional changes.

Reply:
No. `ndc-test` needs to be updated alongside `ndc-models` and `ndc-sdk`, and it uses `reqwest` `12`, which is the reason for this change.